static/src/javascripts-legacy/projects/common/modules/experiments/tests/epic-for-brexit-cohort.js

```diff
@@ -17,8 +17,8 @@ define([
 ) {
 
     function hasBrexitTag(){
-        var tags = config.page.keywordIds.concat(config.page.nonKeywordTagIds);
-        return tags.indexOf('politics/eu-referendum') > -1;
+        var tags = config.page.keywordIds && config.page.keywordIds.concat(config.page.nonKeywordTagIds);
```

Comment:
There is also a `get()` method, which gives you safe access. Never used it, but it should be possible to do:

```js
config.get('page.keywordIds', []).concat()
```